URBNSwiftAlert/Classes/URBNSwAlertStyler.swift

```diff
@@ -0,0 +1,421 @@
+//
+//  URBNSwAlertStyler.swift
+//  Pods
+//
+//  Created by Kevin Taniguchi on 5/22/17.
+//
+//
+
+import Foundation
+import URBNConvenience
+
+public struct URBNSwAlertStyler {
+    
+    /**
+     * Pass no to disable blurring in the background
+     */
+    public var blurEnabled = true
+    
+    /**
+     * Background color of alert view
+     */
+    public var backgroundColor = UIColor.white
+    
+    /**
+     * Tint color of the view behind the Alert. Blur must be disabled
+     */
+    public var backgroundTintColor: UIColor?
+    
+    /**
+     * Radius of the blurred snapshot
+     */
+    public var blurRadius: CGFloat = 5.0
+    
+    /**
+     * Opacity of the alert view's shadow
+     */
+    public var alertViewShadowOpacity: Float = 0.0
```

Comment:
Similar to the blur example, you could create a Shadow type and encapsulate all these shadow properties.

I was going to suggest using NSShadow, but UIView.layer actually has more shadow properties than NSShadow supports.